templates/taxi/driver_confirm_delete.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Delete driver?</h1>
+  <form action="" method="post">
+    {% csrf_token %}
+
+    <input type="submit" value="Yes" class="btn btn-danger">

```

Comment:
Add a `Cancel` button apart from the `Delete` one as mentioned in the checklist. Reattach a corresponding screenshot

Reply:
it's already done. Line 9